app/player.rb

```diff
@@ -36,10 +44,22 @@ def collision_points
     [@top_aft, @bottom_aft, @nose]
   end
 
+  def remove_life
+    if @state != :inmune
+      @lifes -= 1 if @lifes > 0
+      @state = :inmune
+      die! if @lifes == 0
+    end
+  end
+
   def die!
     @state = :dead
   end
 
+  def lifes
+    return @lifes.to_s
```

Comment:
It makes more sense to return `@lifes` as a number here. Also, you can replace this by `attr_reader :lifes` at the beginning of the file.